.vscode/tasks.json

```diff
@@ -1,16 +1,24 @@
 {
 	"version": "0.1.0",
-	"command": "gulp",
-	"isShellCommand": true,
 	"tasks": [
 		{
+			"taskName": "build",
+			"command": "npm",
+			"isShellCommand": true,
+			"args": ["run", "compile"],
+			"showOutput": "always",
+			"isBuildCommand": true
+		},
+		{
 			"taskName": "test",
 			"showOutput": "always",
 			"isTestCommand": true

```

Comment:
Is the 'test' command still correct? I would have thought it would need a `command` element now that it is gone from the root.